### PR TITLE
build: add a BOM to the project

### PR DESCRIPTION
**Pull request by timyates**
No versions updated, they will be done in a new release

This is just for a patch release

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,7 +1,11 @@
 projectVersion=2.1.2-SNAPSHOT
+projectGroup="io.micronaut.rxjava3"
```
Comment (sdelamo) [inline gradle.properties:2]: I think the quotes should be removed

**Review by sdelamo — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (timyates) [reply gradle.properties]: Gone 👍

**Review by sdelamo — APPROVED** [review]

**Comment by sdelamo** [comment]
> This is just for a patch release

@timyates we cannot release a new module in a patch release. This should be in a minor release of rx-java3. We can do that tomorrow.

**Comment by timyates** [comment]
@sdelamo Cool, thanks 👍 

Question for tomorrow, should I have updated 

```
githubCoreBranch=3.2.x
```

In `gradle.properties`?

See you tomorrow

**Comment by sdelamo** [comment]
> @sdelamo Cool, thanks 👍
> 
> Question for tomorrow, should I have updated
> 
> ```
> githubCoreBranch=3.2.x
> ```
> 
> In `gradle.properties`?
> 
> See you tomorrow

We will need to merge these PRs: 

https://github.com/micronaut-projects/micronaut-rxjava3/milestone/10

File: gradle.properties [hunk 2]
```diff
@@ -15,5 +19,3 @@ developers=Graeme Rocher
 
 githubCoreBranch=3.2.x
 bomProperty=micronautRxJava3Version
-bomProperties=rxJava3Version
```
Comment (timyates) [inline gradle.properties:18]: @sdelamo I removed `bomProperties` Just checking this is correct?

Comment (sdelamo) [reply gradle.properties:18]: Yes, I think it is correct

**Pull request merged**